runtime/autoload/health/provider.vim

```diff
@@ -487,9 +487,71 @@ function! s:check_ruby() abort
   endif
 endfunction
 
+function! s:check_node() abort
+  call health#report_start('Node provider (optional)')
+
+  let loaded_var = 'g:loaded_node_provider'
+  if exists(loaded_var) && !exists('*provider#node#Call')
+    call health#report_info('Disabled. '.loaded_var.'='.eval(loaded_var))
+    return
+  endif
+
+  if !executable('node') || !executable('npm') || !executable('yarn')
```

Comment:
why is yarn a requirement, @billyvg ?

Reply:
Oops, it shouldn't be, it seems I didn't pull in https://github.com/neovim/node-host/commit/40c370a0590a20b740b278bf584c7765bcd0df28